subgraph/src/datasources/scannerregistry.ts

```diff
@@ -1,77 +1,223 @@
+import eventId from "../utils/event";
+import transactionLog from "../utils/transaction";
 import {
-    ScannerManager,
-    ScannerTransfer,
-    ScannerManagerEnabled,
-    ScannerEnabled,
-} from '../../generated/schema'
+  ScannerUpdated as ScannerUpdatedEvent,
+  Transfer as TransferEvent,
+  ScannerEnabled as ScannerEnabledEvent,
+  ManagerEnabled as ManagerEnabledEvent,
+} from "../../generated/ScannerRegistry/ScannerRegistry";
 
 import {
-    ManagerEnabled as ManagerEnabledEvent,
-    ScannerEnabled as ScannerEnabledEvent,
-    Transfer       as TransferEvent,
-} from '../../generated/ScannerRegistry/ScannerRegistry'
+  ScannerEnabled,
+  ScannerManager,
+  ScannerManagerEnabled,
+  ScannerTransfer,
+  ScannerUpdated,
+} from "../../generated/schema";
+import { fetchAccount } from "../fetch/account";
+import { fetchScanner } from "../fetch/scanner";
+import { newMockEvent } from "matchstick-as";
+import { Address, BigInt, ethereum } from "@graphprotocol/graph-ts";
 
-import {
-    events,
-    transactions,
-} from '@amxx/graphprotocol-utils'
+export function handleScannerUpdated(event: ScannerUpdatedEvent): void {
+  let scanner = fetchScanner(event.params.scannerId);
+  scanner.metadata = event.params.metadata;
+  scanner.chainId = event.params.chainId;
+  scanner.save();
 
-import { fetchAccount } from '../fetch/account'
-import { fetchScanner } from '../fetch/scanner'
+  const ev = new ScannerUpdated(eventId(event));
+  ev.transaction = transactionLog(event).id;
+  ev.timestamp = event.block.timestamp;
+  ev.scanner = scanner.id;
+  ev.metadata = event.params.metadata;
+  ev.chainId = event.params.chainId;
+  ev.save();
+}
 
 export function handleTransfer(event: TransferEvent): void {
-    let from    = fetchAccount(event.params.from)
-    let to      = fetchAccount(event.params.to)
-    let scanner = fetchScanner(event.params.tokenId)
-    scanner.owner = to.id
-    scanner.save()
-
-    let ev = new ScannerTransfer(events.id(event))
-    ev.transaction = transactions.log(event).id
-    ev.timestamp   = event.block.timestamp
-    ev.scanner     = scanner.id
-    ev.from        = from.id
-    ev.to          = to.id
-    ev.save()
+  let scanner = fetchScanner(event.params.tokenId);
+  let from = fetchAccount(event.params.from);
+  let to = fetchAccount(event.params.to);
+  scanner.owner = to.id;
+  scanner.save();
+
+  const ev = new ScannerTransfer(eventId(event));
```

Comment:
In some places you use `const`, in other you use `let` (sometimes within the same file).